wicker/core/storage.py

```diff
@@ -72,6 +82,17 @@ def check_exists_s3(self, input_path: str) -> bool:
         except ClientError:
             return False
 
+    @retry(Exception, tries=get_config().s3_storage_config.retries, backoff=5, delay=4, logger=logger)
```

Comment:
Could we also have backoff and delay in configs too?

Reply:
Done.